TeknoParrotUi.Common/GameProfiles/KingofFightersMaximumImpactRegulationA.xml

```diff
@@ -1,14 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <GameProfile xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <GameName>KOF Maximum Impact Regulation A</GameName>
+  <GameName>KOF Maximum Impact - Regulation A</GameName>
```

Comment:
no - in official name

Reply:
It should, because it's a version name subtitle.

Reply:
> it should
but it doesn't
If you wanted to be *really* picky about game titles, I believe it should be ``KOF Maxiimum Impact Regulation "A"``? That's the name on PS store, so I'm assuming that's the official title.

Reply:
https://game.snk-corp.co.jp/official/kof-mi-ra/